cmd/srcd/cmd/init_test.go

```diff
@@ -31,20 +33,15 @@ func TestInitTestSuite(t *testing.T) {
 }
 
 func (s *InitTestSuite) SetupTest() {
-	dir, err := filepath.Abs(filepath.Join("..", "..", "..", ".integration-testing"))
+	var err error
+	s.testDir, err = ioutil.TempDir("", "init-test")
```

Comment:
👍

Reply:
I'm gonna try this first locally coz if I remember well I also did like this initially, but then the folder was created in a path that docker wasn't able to mount (on OSX).

Reply:
Yup, unfortunately it has some problems on OSX:

<img width="1440" alt="Screenshot 2019-03-12 at 14 23 59" src="https://user-images.githubusercontent.com/5599208/54203403-85caeb80-44d2-11e9-8a8b-86e2076e5fe1.png">

In my case:

```
✘  se7entyse7en in ~/Projects/.go-workspace/src/github.com/src-d/engine (carlosms-integration-init)  $ echo $TMPDIR                                                              [03/12/19 | 14:23:46.790]
/var/folders/80/4kkg57_13jl8wyslvlzdfhr80000gn/T/
```

If we want to use `ioutil.TempDir` maybe we could set the env var to another location.

Reply:
Is it enough to do this in our makefile?
```
TMPDIR="./integration-test-tmp"

test-integration-no-build:
	rm -rf $(TMPDIR)
	mkdir $(TMPDIR)
	$(GOTEST_INTEGRATION) github.com/src-d/engine/cmd/srcd/cmd/

test-integration: clean build docker-build test-integration-no-build
```

Reply:
coud we add the `rm -rf $(TMPDIR)` to the `clean` target?, and use it as `test-integration-no-build` dependency?

Reply:
```test-integration-no-daemon: clean-integration build-integration
	TMPDIR=`pwd`/integration-test-tmp
	rm -rf $(TMPDIR)
	mkdir $(TMPDIR)
	$(GOTEST_INTEGRATION) github.com/src-d/engine/cmd/srcd/cmd/
```

Reply:
The problem is that we should restore the original value of `TMPDIR`, as the user doesn't expect it to be changed.

Reply:
I saw `$(PWD)` and `$(shell pwd)`, but never `'pwd'`
I wonder if that's what is causing your problem @se7entyse7en

Reply:
@dpordomingo you're right! This worked and didn't override the env var once it finished. 👍 

```TMPDIR=$(PWD)/integration-test-tmp

test-integration-no-daemon: clean-integration build-integration
	rm -rf "$(TMPDIR)"
	mkdir "$(TMPDIR)"
	$(GOTEST_INTEGRATION) github.com/src-d/engine/cmd/srcd/cmd/
```

Reply:
Since it's not straightforward, and we already have some test in master using `ioutil.TempDir`, would you agree to keep this PR working for linux, and find a proper and clean solution in a different PR later? If you agree I'll open a new issue with a link to this discussion.

Reply:
it's ok for me